docs/pages/lunadefend/cli/start.md

```diff
@@ -18,19 +18,21 @@ sidebar_position: 2
   ~
 -->
 
-The `lunasec start --env <environment>` command brings up the LunaSec stack in the provided environment configuration. 
```

Comment:
Sorry if I missed it and it's in here but let's add a one liner right when the CLI is first introduced..probably in cli/installation.md that says "This cli is called `lunasec` for legacy reasons.  It was created before we renamed the tool to LunaDefend"  

I know we were talking about a dedi file for that but it seems much simpler to do a quick one liner right on the CLI to me.  Much more likely to be read.